services/src/main/java/org/keycloak/services/resources/admin/UsersResource.java

```diff
@@ -104,17 +105,20 @@ public UsersResource(RealmModel realm, AdminPermissionEvaluator auth, AdminEvent
     @Consumes(MediaType.APPLICATION_JSON)
     public Response createUser(final UserRepresentation rep) {
         auth.users().requireManage();
-
+        String username = rep.getUsername();
+        if (username == null || username.length() == 0) {
```

Comment:
`username.trim().length() == 0` maybe if you don't want blank user

Reply:
Just as you said. I will fix so.